### DISCUSSION TITLE
Rustdesk Server on Plesk / Docker

### DISCUSSION THREAD

**Discussion by kornblumenapotheke** [rustdesk/rustdesk#8413] [Q&A]
Hello,
I am struggling on instaling the dockerimage on plesk.
when I install as shown here 
https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/docker/
no ports are exposed.

Can please someone guide me.

**Comment by rustdesk** [comment]
Nobody will answer you here, please go to https://discord.com/invite/nDceKgxnkV

**Comment by xlionjuan** [comment]
https://youtu.be/bKFMS5C4CG0

**Comment by irzyxel** [comment]
https://sjoker.net/2024/12/17/selfhost-rustdesk-using-docker-and-plesk/
